PhysicsTools/HeppyCore/python/framework/eventstfile.py

```diff
@@ -22,7 +22,9 @@ def size(self):
 
     def to(self, iEv):
         '''navigate to event iEv.'''
-        self.tree.GetEntry(iEv)
+        nbytes = self.tree.GetEntry(iEv)
+        if nbytes < 0:
+            raise IOError("Could not read file")
```

Comment:
Hi Joosep, Could you please add the file path to the error message? This will make it easier for users to find out what's going wrong.